test/end-to-end/vehicle.js

```diff
@@ -58,6 +58,10 @@ test('vehicle unlock', async(t) => {
   await t.notThrowsAsync(t.context.vehicle.unlock());
 });
 
+test('vehicle batch', async(t) => {
+  await t.notThrowsAsync(t.context.vehicle.batch(['/odometer']));
```

Comment:
```suggestion
  await t.notThrowsAsync(t.context.vehicle.batch(['/odometer', '/location']));
```

Reply:
Throw in another endpoint to make sure we can hit more than 1 with batch :smile: